assignments/prototypes.js

```diff
@@ -7,14 +7,23 @@
   
   Each constructor function has unique properites and methods that are defined in their block comments below:
 */
-  
+
 /*
   === GameObject ===
   * createdAt
   * dimensions
   * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
 */
 
+function GameObject(createdAt, dimensions) {
```

Comment:
You have the right logic here but look at how you are creating a mage, swordsman, and archer. You are passing in an object. So for these prototypes, you'd have to pass in an object and then target each item. 

For example, if you passed an object to GameObject you'd target `createdAt` as such:
```
function GameObject(obj) {
  this.createdAt = obj.createdAt;
}
```